tests/test_sft_trainer.py

```diff
@@ -381,15 +381,17 @@ def test_run_causallm_lora_and_inference(request, target_modules, expected):
 
 
 ############################# Finetuning Tests #############################
-
-
 def test_run_causallm_ft_and_inference():
     """Check if we can bootstrap and finetune tune causallm models"""
+    _test_run_causallm_ft_and_inference(TRAIN_ARGS, MODEL_ARGS, DATA_ARGS)
+
+############################# Helper functions #############################
+def _test_run_causallm_ft_and_inference(training_args=TRAIN_ARGS, model_args=MODEL_ARGS, data_args=DATA_ARGS):
     with tempfile.TemporaryDirectory() as tempdir:
-        train_args = copy.deepcopy(TRAIN_ARGS)
+        train_args = copy.deepcopy(training_args)
```

Comment:
I guess you are not passing them through function in the other test cases. Do you want to make all of them consistent with one approach?